packages/server/src/AppConfig.ts

```diff
@@ -0,0 +1,3 @@
+export const APIKEYS_STORAGE_JSON = !process.env.APIKEY_STORAGE_TYPE || process.env.APIKEY_STORAGE_TYPE === 'json'
+export const APIKEYS_STORAGE_DB =
+    process.env.APIKEY_STORAGE_TYPE === 'db' || process.env.APIKEY_STORAGE_TYPE === 'DB' || process.env.APIKEY_STORAGE_TYPE === 'Db'
```

Comment:
should we make this more geneic?

```
export const appConfig = {
    apiKeys: {
        storageType: process.env.APIKEY_STORAGE_TYPE ? process.env.APIKEY_STORAGE_TYPE.toLowerCase() : 'json',
    }
    // todo: add more config options here like database, log, storage, credential and allow modification from UI
}
```

Reply:
Also, this new env variable needed to add at other places like `start.ts`, `env.example`, etc

Reply:
> Also, this new env variable needed to add at other places like `start.ts`, `env.example`, etc

changes pushed

Reply:
Changes pushed!!!